enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/items/OCIItem.java

```diff
@@ -27,11 +27,12 @@
  * 
  * @author Jan Horvath
  */
-public abstract class OCIItem {
+public abstract class OCIItem implements NamedReference {
     final OCID id;
     final String name;
     final String compartmentId;
     String description;
+    private String referenceName = "DEFAULT";
```

Comment:
Q: each `OCI item` becomes a valid `NamedReference` (regadless of OCI item type ?)